Update to 1.16

specificlanguage opened the pull request:
Updated:
- VoxelMap
- Litematica (has new dependency)
- BetterPvP
- Optifine (currently on preview, be warned)
- Optifabric
- Grid (adds Forge support)
- Mousewheelie
- MiniHUD
- malilib (spelled incorrectly in previous versions)
- BetterSprinting
- WorldEdit
- FabricAPI

Lost:
- AdvancedMacros
- Ghostwriter

squareblob committed:
hmmm

squareblob committed:
hmmm

squareblob committed:
testing

squareblob committed:
Delete voxelmap.properties

squareblob committed:
Delete voxelmap.properties

squareblob committed:
testing

squareblob committed:
testing

squareblob committed:
Delete minihud.json

squareblob committed:
testing

squareblob committed:
Delete voxelmap.properties

squareblob committed:
Delete voxelmap.properties

squareblob committed:
Delete minihud.json

squareblob committed:
Delete options.txt

squareblob committed:
Add very shitty prototype

squareblob committed:
Create README.md

squareblob committed:
update mod files

squareblob committed:
update mod files

squareblob committed:
Update README.md

squareblob committed:
updates

squareblob committed:
updates

squareblob committed:
Update README.md

squareblob committed:
i forgot this existed

squareblob committed:
Merge branch '1.14' of github.com:squareblob/Civ-Mods into 1.14

specificlanguage committed:
Update to 1.16

Updated:
- VoxelMap
- Litematica (has new dependency)
- BetterPvP
- Optifine (currently on preview, be warned)
- Optifabric
- Grid (adds Forge support)
- Mousewheelie
- MiniHUD
- malilib (spelled incorrectly in previous versions)
- BetterSprinting
- WorldEdit
- FabricAPI

Lost:
- AdvancedMacros
- Ghostwriter

Gjum commented:
> Optifine/optifabric

I read Sodium is what the cool kids use these days?

there's also MyBrightness for gamma

and xaero's map is also 1.16 already

specificlanguage committed:
More mods added

Sodium - Optifine 2
Phosphor - Lighting optimizer
Xaero's Minimap - by popular^TM request

MyBrightness wasn't added since it was built for a 1.16 preview, so for safekeeping I didn't keep it
